### PR TITLE
Feat/add travis

### PR DESCRIPTION
**Pull request by rmi22186**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -31,7 +31,8 @@
     "rollup-plugin-node-resolve": "^5.0.3",
     "shelljs": "^0.8.3",
     "should": "^7.1.0",
-    "watchify": "^3.9.0"
+    "watchify": "^3.9.0",
+    "mparticle-sdk-javascript": "github:mparticle/mparticle-sdk-javascript"
```
Comment (samdozor) [inline package.json]: shouldn't this be mparticle-web-sdk and pull from NPM?

File: test/boilerplate/test-index.js [hunk 2]
```diff
@@ -1,26 +1,8 @@
...
-    var self = this;
-
-    this.addForwarder = function (forwarder) {
-        self.forwarder = new forwarder.constructor();
-    };
-
-    this.getCurrentUser = function() {
-        return currentUser();
-    };
+require('../../node_modules/mparticle-sdk-javascript/src/main.js');
```
Comment (samdozor) [inline test/boilerplate/test-index.js]: mparticle-web-sdk?

File: .travis.yml [hunk 3]
```diff
@@ -2,7 +2,7 @@ language: node_js
 node_js:
   - "7"
```
Comment (samdozor) [inline .travis.yml]: node 7 is pretty old now...maybe better to leave out the version so it just always uses the latest stable?

**Comment by rmi22186** [comment]
Hey - the PR is actually quite out of date, I’ll close it.

**Pull request closed without merging**